ui/app/components/switch.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @module Switch
```

Comment:
out of curiosity, why'd you choose the name `Switch` and not `Toggle`? i know anecdotally the design + our team has referred to these elements as toggles. not a huge issue, but it could cause some confusion

Reply:
I chose that name based on the css class that the existing toggle has, but I am absolutely not opposed to switching it to Toggle! The only other consideration here is that there are a couple other `toggle` components (b64Toggle, ToggleButton) out there that act more like buttons, but being able to see the differences in StoryBook makes that a nonissue in my mind